frontend/src/pages/Player/Toolbar/DevToolsWindowV2/ConsolePage/style.css.ts

```diff
@@ -5,6 +5,9 @@ import { recipe } from '@vanilla-extract/recipes'
 export const consoleBox = style({
 	paddingLeft: 8,
 	height: '100%',
+	display: 'flex',
+	flexDirection: 'row',
+	justifyContent: 'center',
```

Comment:
this still isn't centered

![Screenshot from 2023-09-05 22-33-31](https://github.com/highlight/highlight/assets/1351531/075de6fd-ea5f-4b99-82b9-ace16d5c694f)

![image](https://github.com/highlight/highlight/assets/1351531/e042d7f6-28f4-4ec0-896b-a6ccac803aea)